saleor/graphql/payment/mutations.py

```diff
@@ -801,9 +801,7 @@ class Arguments:
         )
 
     class Meta:
-        description = (
-            "Create transaction for checkout or order." + ADDED_IN_34 + PREVIEW_FEATURE
-        )
+        description = "Create transaction for checkout or order." + ADDED_IN_34
```

Comment:
I am not convinced that we should remove the PREVIEW label from transactionCreate and transactionUpdate. Although they were added in version 3.4, we are introducing some breaking changes in version 3.13 based on the assumption that both of these mutations are in preview state. By "breaking changes," I mean that in version 3.13, we accept both new and old flows, but in version 3.14, the transactionAPI will only work with the flow introduced in version 3.13.
IMHO We should wait to remove the preview label from TransactionAPI until version 3.14/3.15.